### PR TITLE
Update links to developercloud

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary

Updating links that point to WDC documentation to go to the Watson
console.

- Update links that go to WDC to point to IBM Cloud
- Update references of Bluemix to IBM Cloud
- Update service landing pages from WDC to newer marketing pages

### CODE REVIEW

**Comment by mediumTaj** [comment]
@SirSpidey can you give this a quick once over to make sure all your changes made it in there?

File: src/IBM.WatsonDeveloperCloud.Discovery.v1/README.md [hunk 1]
```diff
@@ -212,4 +212,4 @@ var result = _discovery.Query(<environmentId>, <collectionId>, <query>);
 
 [discovery]: https://www.ibm.com/watson/developercloud/discovery.html
 [discovery-sizing]: https://www.ibm.com/watson/developercloud/discovery.html#pricing-block
```
Comment (SirSpidey) [inline src/IBM.WatsonDeveloperCloud.Discovery.v1/README.md]: https://www.ibm.com/watson/services/discovery/#pricing-block

**Comment by mediumTaj** [comment]
Ah ok what do the links in the `IDiscoveryService.cs` and `IPersonalityInsightsService.cs` need to change to? These are generated from the api explorer so they will probably need to change there as well. I don't think I found any you didn't, it may have been corrected in regenerating the services.

**Comment by SirSpidey** [comment]
@mediumTaj if those links in the .cs files are generated, we can handle that in changes to the API explorer then. But the changes are described here:

- **IDiscoveryService.cs**: https://github.com/watson-developer-cloud/dotnet-standard-sdk/pull/167/files#diff-f85cf7c85d3fa1596c7ed183b66750a3R191
- **IPersonalityInsightsService.cs**: https://github.com/watson-developer-cloud/dotnet-standard-sdk/pull/167/files#diff-eecf43cbfe3937155614f66244951728R29

**Pull request merged**